src/collar/oc_particle.lsl

```diff
@@ -155,7 +155,7 @@ FindLinkedPrims() {
 
     for (i=-1; i<=linkcount;++i) {
         string sPrimName = llToLower(llStringTrim(llList2String(llGetLinkPrimitiveParams(i,[PRIM_NAME]),0),STRING_TRIM));
-        if (llToLower(sPrimName) == "leashpoint" || llToLower(sPrimName) == "ooc") {
```

Comment:
Are you sure we don't need the llToLower?  I have a vague memory of code like this being used to account for uppercased menu buttons.

Reply:
take a look at line 157 - there is already    sPrimName = llToLower(